### ISSUE TITLE
Secondary bibliography for mss

### ISSUE THREAD

**Issue by eu-genia** [BetaMasaheft/Documentation#2431]
Historical mss sould have a list of bibliography they were mentioned in (as underlined by @abausi on 6-10-2023).
We schould explore the possibility to
- in Zotero, add mss identifiers as tags for relevant publications
- have an API link with a variable in xquery to automatically extract the relevant bibliography

**Comment by eu-genia** [comment]
Now inserted a link and tested for ESum039 https://betamasaheft.eu/manuscripts/ESum039/main at the bottom of the record 
![image](https://github.com/BetaMasaheft/Documentation/assets/17987728/9bb5e181-2d0a-45d9-a9ef-1fbe8ef25475)

(without embedding for the moment)

It is now possible in Zotero among tags add any BM IDs (mss, places, persons, etc, and get the bibliography automatically extracted in this way)

What do you think @thea-m @abausi @DenisNosnitsin1970 @CarstenHoffmannMarburg

**Comment by CarstenHoffmannMarburg** [comment]
Does this mean, we should add the tags of the manuscripts to the Zotero record, whenever we come across any mentioning of manuscripts in any secondary literature or is it optional?

**Comment by thea-m** [comment]
It looks good and make sense to me. I expect people not knowing the history behind this function to be possibly confused by the two different places to check bibliography (and disappointed that the new one will only slowly start to show results). Maybe it can be a "highlight of the month" and also be explained informally in a centre meeting to keep the communication and decisions behind new features transparent and understandable. It should also be clear that only Bm employees are expected to carry out this additional step.

**Comment by eu-genia** [comment]
We can explore how this works. 
Since @abausi has suggested that secondary bibliography for historical manuscripts should not be neglected I suggest that whenever we have an article that speaks about mss it would be desirable to add the IDs among the Zotero tags (and we can also add IDs for all other types of entities if desired).
I assume it is simpler and faster for many to add a tag in Zotero than to do the proper editing in XML so lists of relevant bibliography can be generated much easier. 
I also think that actually anyone, not only Bm employees, are welcome to tag bibliography records in Zotero.

**Comment by eu-genia** [comment]
(if we are then satisified with the way it works we could also say that this is the only way for us to deal with secondary bibliography, and then we eventually suppress the current XML bits for secondary bibliography for all records and embed the Zotero-generated list, then we only have it once)

**Comment by DenisNosnitsin1970** [comment]
We can try, but do I understand that physically the indications to the bibliography will not be in XML file, then? And I would see all bibliographical items only from the surface of the app, but not in in the XML file, right? And how can we make a reference to exact pages or numbers through Zotero?
I would have Zotero as an additional way of handling the bbliography, but I am not sure that we should switch ot this way completely. I personally would prefer to have everything in one location, not in two (consider to the fact that for one manuscript we already work in several repositories). It seems to me that,  to include a few bibliographic  lines in an XML file is not a bigger work than to tag an Item in Zotero. Even though, I might indeed find it is useful to tag an item in Zotero as related to one or another manuscript, but perhaps as temporary indication, not as the main tool (I personally would transfer this infor to the main XML file, at the end).
I routinely include items of secondary literature into the descriptions, the main issue is not so much to encode the title, but to find some at the right moment :)

**Comment by eu-genia** [comment]
> I routinely include items of secondary literature into the descriptions, the main issue is not so much to encode the title, but to find some at the right moment :)

this exactly was the reason to attempt to invert the perspective and do the cataloguing on the other end as well.

**Comment by thea-m** [comment]
I completely agree that anyone is welcome to tag bibliography records in Zotero. I meant expectations, not possibilities. In my experience anything vaguely Bm-related (such as finding IDs) understandably often takes much longer for anyone outside of the project, in ways often difficult for us to predict. 
My main question is the same as already asked by Denis: How could we in tagging only in Zotero make references to precise page ranges?

**Comment by abausi** [comment]
Further on this, how to start to collect relevant bibliography on MSS we deal witz: my (quite feasible) suggestion: hire a HiWi and drag all vbibliography on MSS from the EAe. This is a relatively easy task, since we have all of them listed, according to clear formal rules, in the "Src." section of each bibliography. I definitely think this would booster the coverage of the corpus. It would be necessary to make a calculation, how long this would take, and who could do this, if any student is available.

**Comment by eu-genia** [comment]
@karljonaskarlsson this was the discussion I mentioned

@JanetFre do you think you could give this a try as a next task?
I have prepared a spreadsheet to fill: https://docs.google.com/spreadsheets/d/13B0t8HKUbT4bEIl9k9sWRFuw193GaLId8b5z2zgbixQ/edit?usp=sharing 

The EAe is avaialble in the [Elektrobibliothek](Z:\aai-hlz-research\ELEKTROBIBLIOTHEK\EAE)

I suggest that the steps are 
1) looking for manuscripts mentioned in the text and in the bibliography of the sources (they are usually prefixed by "ms." or "mss.", unfortunately not always, at least for EMML as far as I can see) and insert the shelfmark as appears in the EAe plus the EAe record for it
2) If possible, list also other bibl if mentioned as describing a manuscript 
3) check if the bibl record exists for the EAe article in Zotero, add the tag - if not, create a Zotero record with the tag of the type bm:AuthorYearEAeKeyword (like bm:Uhlig2003EAeAbaqte) and add the Zotero tag; same for other possible references
3) As a separate step the shelfmarks should be mapped to the existing Bm ID for manuscripts

I am not sure that this is an easy task but let us try! (Would have been easier if we had had already an index of manuscripts quoted in EAe as I did for COMSt, but I did not do it back then)

I started now from EAE I up to page 42, you can try to retrace my steps from the file and the spreadsheet and continue from there. Thank you!

**Comment by JanetFre** [comment]
Sure, I will give it a go.

**Comment by eu-genia** [comment]
@e-schwarz thank you liza for having a look too!
@JanetFre Liza Schwarz will be helping too. Maybe we can divide the volumes for a faster progress )
